code/game/gamemodes/cult/talisman.dm

```diff
@@ -278,11 +259,145 @@ Rite of Disorientation
+	color = "#33cc33" // green
+	invocation = "N'ath reth sh'yro eth draggathnor!"
+
+/obj/item/weapon/paper/talisman/armor/invoke(mob/living/user, successfuluse = 1)
+	. = ..()
 	user.visible_message("<span class='warning'>Otherworldly armor suddenly appears on [user]!</span>", \
 						 "<span class='cultitalic'>You speak the words of the talisman, arming yourself!</span>")
 	user.equip_to_slot_or_del(new /obj/item/clothing/head/culthood/alt(user), slot_head)
 	user.equip_to_slot_or_del(new /obj/item/clothing/suit/cultrobes/alt(user), slot_wear_suit)
 	user.equip_to_slot_or_del(new /obj/item/clothing/shoes/cult/alt(user), slot_shoes)
 	user.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/cultpack(user), slot_back)
+	user.drop_item()
 	user.put_in_hands(new /obj/item/weapon/melee/cultblade(user))
+	user.put_in_hands(new /obj/item/weapon/restraints/legcuffs/bola/cult(user))
+
+/obj/item/weapon/paper/talisman/armor/attack(mob/living/target, mob/living/user)
+	if(iscultist(user) && iscultist(target))
+		user.drop_item()
+		invoke(target)
+		qdel(src)
+		return
+	..()
+
+//Talisman of Horrors: Breaks the mind of the victim with nightmarish hallucinations
+/obj/item/weapon/paper/talisman/horror
+	cultist_name = "Talisman of Horrors"
+	cultist_desc = "A talisman that will break the mind of the victim with nightmarish hallucinations."
+	color = "#ffb366" // light orange
+	invocation = "Lo'Nab Na'Dm!"
+
+/obj/item/weapon/paper/talisman/horror/attack(mob/living/target, mob/living/user)
+	if(iscultist(user))
+		user.visible_message("<span class='cultitalic'>You disturb [target] with visons of the end!</span>")
+		if(iscarbon(target))
+			var/mob/living/carbon/H = target
+			H.reagents.add_reagent("mindbreaker", 25)
+		qdel(src)
+
+//Talisman of Fabrication: Creates a construct shell out of 25 metal sheets.
+/obj/item/weapon/paper/talisman/construction
+	cultist_name = "Talisman of Construction"
+	cultist_desc = "Use this talisman on at least twenty-five metal sheets to create an empty construct shell"
+	invocation = "Ethra p'ni dedol!"
+	color = "#000000" // black
+
+/obj/item/weapon/paper/talisman/construction/attack_self(mob/living/user)
+	if(iscultist(user))
+		user << "<span class='warning'>To use this talisman, place it upon a stack of metal sheets.</span>"
+	else
+		user << "<span class='danger'>There are indecipherable images scrawled on the paper in what looks to be... <i>blood?</i></span>"
+
+
+/obj/item/weapon/paper/talisman/construction/attack(obj/M,mob/living/user)
+	if(iscultist(user))
+		user << "<span class='cultitalic'>This talisman will only work on a stack of metal sheets!</span>"
+		log_game("Construct talisman failed - not a valid target")
+		
+/obj/item/weapon/paper/talisman/construction/afterattack(obj/item/stack/sheet/metal/target, mob/user, proximity_flag, click_parameters)
+   	if(proximity_flag && istype(target) && iscultist(user))
+		..()
+		if(istype(W, /obj/item/weapon/paper/talisman/construction))
```

Comment:
This line is unneeded; remove it and lower the indentation on the following lines.